Formula/sqlcipher.rb

```diff
@@ -15,6 +15,9 @@ class Sqlcipher < Formula
 
   depends_on "openssl@1.1"
 
+  uses_from_macos "tcl-tk" => :build
```

Comment:
While this gets us past the build error, there is now a contradiction here because we also pass `--disable-tcl` as an argument.  It's fine to add this for now so we can resolve any other errors, but we should either remove that build argument, or figure out how to actually disable TCL so it's not needed as a dependency.